app/repositories/orderrepository.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace Repositories;
+
+use Models\Order;
+
+class OrderRepository extends Repository {
+
+    public function getAllOrders() {
+        try {
+            $sql = "SELECT * FROM Orders";
+            $stmt = $this->connection->prepare($sql);
+            $stmt->execute();
+            $ordersData = $stmt->fetchAll();
+            
+            if ($ordersData === false) {
```

Comment:
het is netter als je `$ordersData === false` verandert naar `!$ordersData` dat vangt meer senenarios op omdat je dan checkt of het een 'nullish' value heeft en niet alleen false